Gruntfile.js

```diff
@@ -23,6 +23,17 @@ module.exports = function(grunt) {
         ext: '.css'
       }
     },
+	  autoprefixer: {
+	    options: {
+				browsers: ['last 2 versions', 'Opera >= 15', 'Chrome >= 4', 'Firefox >= 16', 'Safari >= 4']
```

Comment:
@XhmikosR `IE >= 10` will not change anything, because -ms prefix is dropped in this (and future) releases of IE.